EIPS/eip-6110.md

```diff
@@ -94,17 +90,15 @@ def parse_deposit_data(deposit_event_data) -> bytes[]:
 def little_endian_to_uint64(data: bytes) -> uint64:
     return uint64(int.from_bytes(data, 'little'))
 
-def event_data_to_deposit_request_rlp(deposit_event_data) -> bytes:
+def event_data_to_deposit_request(deposit_event_data) -> bytes:
     deposit_data = parse_deposit_data(deposit_event_data)
     pubkey = Bytes48(deposit_data[0])
     withdrawal_credentials = Bytes32(deposit_data[1])
-    amount = little_endian_to_uint64(deposit_data[2])
+    amount = deposit_data[2]   # 8 bytes uint64 LE
     signature = Bytes96(deposit_data[3])
-    index = little_endian_to_uint64(deposit_data[4])
+    index = deposit_data[4]    # 8 bytes uint64 LE
 
-    return DEPOSIT_REQUEST_TYPE + rlp([
-        pubkey, withdrawal_credentials, amount, signature, index
-    ])
+    return DEPOSIT_REQUEST_TYPE + pubkey + withdrawal_credentials + amount + signature + index
```

Comment:
Do we need to prepend the type? My understanding is that each element does not have the type, and the type is only used when calculating the commitment hash.

Reply:
Ah just realised this has changed in other PRs :)